zabbix_metrics_watcher.py

```diff
@@ -214,14 +217,41 @@ def run(zabbix_host, osbs_master, config, instance):
 
             logger.info("Found build %s in %s", build_name, status)
             build = Build(build_name, cmd_base)
-            if status == 'Pending':
+            if status == 'New':
+                now = datetime.datetime.now()
+                if build_name not in builds_in_new:
+                    builds_in_new[build_name] = now
+                spent_in_new = [(now - start).total_seconds() for start in builds_in_new.values()]
```

Comment:
I wonder if `spent_in_new = ...` to `_send_zabbix_message` should be in the `status != 'New'` case instead. Also, I still don't really think `STUCK_IN_NEW_THRESHOLD_SECS` is useful, and we should never report 0 for this item if it isn't the case.